frameworks/elastic/build-kibana.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
```

Comment:
Does it get called from the CI test script, test.py? I assume it would have to in order for the kibana tests to run against the kibana stub-universe.